libs/input-mapping/src/Helper/FakeDevStorageTableRewriteHelper.php

```diff
@@ -16,14 +18,23 @@ public function rewriteTableOptionsSources(
         InputTableOptionsList $tablesDefinition,
         ClientWrapper $clientWrapper,
         LoggerInterface $logger
-    ): InputTableOptionsList {
+    ): RewrittenInputTableOptionsList {
+        $newTables = [];
         foreach ($tablesDefinition->getTables() as $tableOptions) {
+            $source = $tableOptions->getSource();
             if ($clientWrapper->hasBranch()) {
-                $tableOptions->setSource($this->rewriteSource($tableOptions->getSource(), $clientWrapper, $logger));
+                $source = $this->rewriteSource($tableOptions->getSource(), $clientWrapper, $logger);
             }
-            $tableOptions->setSourceBranchId($clientWrapper->getDefaultBranch()['branchId']);
+            $sourceBranchId = $clientWrapper->getDefaultBranch()['branchId'];
+            $tableInfo = $clientWrapper->getTableAndFileStorageClient()->getTable($source);
```

Comment:
Tady se naopak proti RealDevStorageTableRewriteHelper používá vždycky stejný klient (reálně vždycky basic, protože je to řízené stejnou featurou, kterou je řízené rozhodování mezi Fake & Real rewriter)